plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -2183,4 +2183,11 @@ public void testAggregationPushdownWithArrays()
                         "(56, VARCHAR 'string_8400', BIGINT '1')," +
                         "(1000, VARCHAR 'string1_8401', BIGINT '1')");
     }
+
+    @Test
+    public void testTableFunctions()
+    {
+        assertQuerySucceeds("SELECT * FROM TABLE(system.query('select * from my_table where city = ''Los Angeles'''))");
+        assertQuerySucceeds("SELECT * FROM TABLE(system.query('select * from mixedCase'))");
```

Comment:
Please verify the correctness of the results and add more test cases:
* DML should fail
* Case sensitivity. e.g. `los angeles` should return an empty result